war/src/main/webapp/help/system-config/master-slave/usage.html

```diff
@@ -6,7 +6,8 @@
       Utilize this node as much as possible
```

Comment:
The initial PR comment listed *utilize* to be replaced by *use*. In this particular case it would require more than just changing the documentation, so may be out of scope.